cider-velox/src/RawDataConvertor.cpp

```diff
@@ -463,10 +471,37 @@ RowVectorPtr RawDataConvertor::convertToRowVector(const CiderBatch& input,
   int num_cols = schema.getColumnCount();
   types.reserve(num_cols);
   columns.reserve(num_cols);
+  int inputColIndex = 0;
+
   for (int i = 0; i < num_cols; i++) {
     ::substrait::Type sType = schema.getColumnTypeById(i);
     types.push_back(getVeloxType(sType));
-    columns.push_back(toVeloxVector(types[i], sType, input.column(i), num_rows, pool));
+    auto currentData = input.column(i);
+    auto columNum = input.column_num();
+    if (sType.kind_case() == substrait::Type::kStruct) {
+      // TODO : (ZhangJie) Support nested struct after cider supported nested (MaYan).
```

Comment:
author name should not present in comments

Reply:
remove it.